.github/actions/setup-deploy/action.yaml

```diff
@@ -0,0 +1,58 @@
+name: "Deploy prerequisites"
+description: "Prerequisites needed to deploy hubs in a 2i2c cluster"
+inputs:
+  provider:
+    description: "Cloud provider a cluster runs on"
+    required: true
+    default: "gcp"
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+
+    - name: Setup gcloud
+      uses: google-github-actions/setup-gcloud@v0
+      with:
+        version: "290.0.1"
+        # This is used for KMS only
+        project_id: two-eye-two-see
+        service_account_key: ${{ secrets.GCP_KMS_DECRYPTOR_KEY }}
+        export_default_credentials: true
+
+    - name: Get gcloud info
+      run: gcloud info
+      shell: bash
+
+    - name: Setup gcloud auth for docker
+      # FIXME: Add more auth providers & registries here as needed
+      run: |
+        gcloud auth configure-docker us-central1-docker.pkg.dev
+      shell: bash
+
+    - name: Setup helm
+      uses: azure/setup-helm@v2.0
```

Comment:
Note that `helm` is bundled in the github environment by default now, they may lag behind a patch version or two, but stay up to date in general.

Not having it here could perhaps save a few seconds and some PRs about bumping this via dependabot etc.

Reply:
Yepp!

Note though that the Python versions are available ahead of time and setup-python makes the chosen version installed in <1 second by changing symlinks, so its a choice of the system-installed version - unless a specific patch version of python is chosen that wasnt available.